.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ orbs:
 jobs:
   test-e2e:
     docker:
-      - image: circleci/node:10-browsers
+      - image: circleci/node:10.15.1-browsers
```

Comment:
It looks like we need a little image name: 10.15.1-jessie or 10.15.1-stretch (https://github.com/CircleCI-Public/circleci-dockerfiles/tree/master/node/images) and then '-browsers'

Reply:
I see!! 👍 let's try this